hopsworks-common/src/main/java/io/hops/hopsworks/common/hdfs/Utils.java

```diff
@@ -91,6 +91,10 @@ public static String getDirectoryPart(String path) {
   public static String getProjectPath(String projectname) {
     return "/" + Settings.DIR_ROOT + "/" + projectname + "/";
   }
+  
+  public static String getDatasetPath(String projectName, String datasetName) {
+    return "/" + Settings.DIR_ROOT + "/" + projectName + "/" + datasetName + "/";
```

Comment:
This needs to be changed. If you pass a feature store or a Hive db to this method, it won't work.

Reply:
fixed
https://github.com/AlexHopsworks/hopsworks/blob/provenance/hopsworks-common/src/main/java/io/hops/hopsworks/common/hdfs/Utils.java#L97